cql3/statements/list_permissions_statement.cc

```diff
@@ -80,9 +80,10 @@ cql3::statements::list_permissions_statement::execute(
         service::query_state& state,
         const query_options& options,
         std::optional<service::group0_guard> guard) const {
-    static auto make_column = [](sstring name) {
+    auto auth_ks = auth::get_auth_ks_name(qp);
+    static auto make_column = [auth_ks = std::move(auth_ks)](sstring name) {
```

Comment:
The lambdas that you modify in this commit are `static`, but you are making them capture `auth_ks` which is calculated dynamically every time the outer method is called. While maybe the auth keyspace name is always the same at the moment, it might not be true after the upgrade procedure is implemented, so IMO this is asking for trouble (and just looking at it without context lights a small red lamp in my head). I think the lambdas should be de-`static`-fied.

Reply:
yes, the same principle as with other static queries (will add lambda to TODO to not forget about it)

Reply:
Given that the migration procedure is now part of the PR -- shouldn't this be fixed?

Reply:
hoped nobody notices :) and I'd be able to do it in a follow up, I can add this commit here if you insist